### DISCUSSION TITLE
How do I add a personal snippet?

### DISCUSSION THREAD

**Discussion by vinoff** [LazyVim/LazyVim#5247] [Q&A]
I am using the default lazyvim config. In the past, I knew how to add a snippet via luasnip, but it seems like luasnip is no longer used..

So, how do I add a personal snippet with blink or friendly-snippets or whatever it is..?

**Comment by dpetka2001** [comment]
You can still use Luasnip if you want. Just enable the Extra. It's just not default. Otherwise just set it up like Luasnip vscode snippets. It's just that the default location of snippets is at `~/.config/nvim/snippets` as this is the default value in blink.

**Comment by dusty-phillips** [comment]
https://lazyvim-ambitious-devs.phillips.codes/course/chapter-14/#_defining_new_snippets
